ftplugin/ps1xml.vim

```diff
@@ -14,6 +14,11 @@ let b:did_ftplugin = 1
 setlocal tw=0
 setlocal commentstring=#%s
 setlocal formatoptions=tcqro
+" Enable autocompletion of hyphenated PowerShell commands,
+" e.g. Get-Content or Get-ADUser
+setlocal iskeyword+=-
```

Comment:
Hi heaths. I wasn't sure about that because I'd never worked with `.ps1xml` files (and I presumed that they should have the same configuration). Now that I've researched what `.ps1xml` files are and how they're used, I realise I was wrong to copy those lines from `.ps1`.

Would it be easier if I resubmit the pull request – or should I just leave it as it is?